subprojects/gst-plugins-bad/sys/msdk/gstmsdkcontext.c

```diff
@@ -474,15 +635,23 @@ gst_msdk_context_get_handle (GstMsdkContext * context)
 #endif
 }
 
+#ifndef _WIN32
 GstObject *
 gst_msdk_context_get_display (GstMsdkContext * context)
 {
-#ifndef _WIN32
   if (context->priv->display)
     return gst_object_ref (GST_OBJECT_CAST (context->priv->display));
-#endif
   return NULL;
 }
+#else
+GstD3D11Device *
+gst_msdk_context_get_device (GstMsdkContext * context)
```

Comment:
How about rename it to get_d3d11_device, and rename gst_msdk_context_get_display to get_va_display.
The name is to generic, I think.

Reply:
fixed, thx.